From d08d2604a9f0f925b93b87d74ee0a2c26c785467 Mon Sep 17 00:00:00 2001
From: Angelos Kolaitis <angelos.kolaitis@canonical.com>
Date: Thu, 14 Jul 2022 18:18:09 +0300
Subject: [PATCH] disable warnings for kubeconfig permissions

---
 cmd/helm/root.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/helm/root.go b/cmd/helm/root.go
index ef92fea9..512f823f 100644
--- a/cmd/helm/root.go
+++ b/cmd/helm/root.go
@@ -206,7 +206,7 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	loadPlugins(cmd, out)

 	// Check permissions on critical files
-	checkPerms()
+	// checkPerms()

 	// Check for expired repositories
 	checkForExpiredRepos(settings.RepositoryConfig)
--
2.25.1
