diff --git a/libcontainer/rootfs_linux.go b/libcontainer/rootfs_linux.go
index b8e792fb..0c045564 100644
--- a/libcontainer/rootfs_linux.go
+++ b/libcontainer/rootfs_linux.go
@@ -107,8 +107,6 @@ func prepareRootfs(pipe io.ReadWriter, iConfig *initConfig) (err error) {

 	if config.NoPivotRoot {
 		err = msMoveRoot(config.Rootfs)
-	} else if config.Namespaces.Contains(configs.NEWNS) {
-		err = pivotRoot(config.Rootfs)
 	} else {
 		err = chroot(config.Rootfs)
 	}
