diff --git a/libcontainer/setns_init_linux.go b/libcontainer/setns_init_linux.go
index 0b9cd885..f6b53d1a 100644
--- a/libcontainer/setns_init_linux.go
+++ b/libcontainer/setns_init_linux.go
@@ -56,11 +56,6 @@ func (l *linuxSetnsInit) Init() error {
 			return err
 		}
 	}
-	if l.config.NoNewPrivileges {
-		if err := unix.Prctl(unix.PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0); err != nil {
-			return err
-		}
-	}
 	if err := selinux.SetExecLabel(l.config.ProcessLabel); err != nil {
 		return err
 	}
