From 76f154cf345ffc7f2b4b9c0e623551c744249d60 Mon Sep 17 00:00:00 2001
From: Angelos Kolaitis <angelos.kolaitis@canonical.com>
Date: Fri, 27 Oct 2023 17:01:33 +0000
Subject: [PATCH] enable cgo

---
diff --git a/Makefile b/Makefile
index c8ced67..0a32341 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ TESTS       := .
 TESTFLAGS   :=
 LDFLAGS     := -w -s
 GOFLAGS     :=
-CGO_ENABLED ?= 0
+CGO_ENABLED := 1
 
 # Rebuild the binary if any of these files change
 SRC := $(shell find . -type f -name '*.go' -print) go.mod go.sum
